.changeset/fluffy-toys-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-css': minor
+---
+
+Adding site-header component.
```

Comment:
Little nit! This goes to the changelog, so perhaps past tense is best?

```suggestion
Created new site-header component.
```

Reply:
Updated, but not the way you suggested. Keeping continuous tense since that's the way we wrote it many times:
https://github.com/microsoft/atlas-design/blob/main/css/CHANGELOG.md

Reply:
Ah, we should probably think about that. I wonder if simple present test is the way to go. Fine to keep on for now though

Reply:
Yeah, we can communicate it and following changesets write in past tense.